storm-server/src/main/java/org/apache/storm/utils/ServerUtils.java

```diff
@@ -214,13 +219,6 @@ public static ClientBlobStore getClientBlobStoreForSupervisor(Map<String, Object
 
     /**
      * Meant to be called only by the supervisor for stormjar/stormconf/stormcode files.
-     *
-     * @param key
-     * @param localFile
-     * @param cb
-     * @throws AuthorizationException
-     * @throws KeyNotFoundException
-     * @throws IOException
```

Comment:
Why are these removed?

Reply:
They have have no description and thus no value. Leaving them empty is refused by checkstyle (which is a good thing imo).